site/src/components/pages/docs/search/index.tsx

```diff
@@ -143,32 +153,30 @@ export const Search: React.VoidFunctionComponent<SearchProps> = ({
     ) : undefined;
 
   return (
-    <>
-      <TextField
-        inputRef={inputRef}
-        onKeyDown={(event: KeyboardEvent<HTMLInputElement>) => {
-          if (searchResults.length > 0) {
-            if (event.key === "ArrowUp") {
-              event.preventDefault();
-
-              if (activeResult === 0) {
-                return setActiveResult(searchResults.length - 1);
-              }
-
-              setActiveResult(activeResult - 1);
-            }
-
-            if (event.key === "ArrowDown") {
-              event.preventDefault();
-
-              if (activeResult > searchResults.length - 1) {
-                return setActiveResult(0);
-              }
+    <Box
+      onKeyDown={(event: KeyboardEvent<HTMLInputElement>) => {
```

Comment:
Nit: arrow down behaves differently than arrow up:

https://user-images.githubusercontent.com/42802102/175256070-ae6857eb-da3d-4fe0-91a0-da2c2121ede5.mov

Reply:
Surprised I didn't catch that, thanks! Addressed in bfc84b2dc36b3d74ea16e1cc7bbb578c707e567f